README.md

```diff
@@ -141,16 +142,18 @@ ansible-playbook job-cleanup.yml                                   \
     --user       $user
 
 ansible-playbook create-db.yml                                     \
-    --extra-vars cluster=$cluster                                  \
+    --extra-vars "cluster=$cluster cluster_type=$cluster_type"     \
     --inventory  ../../concent-deployment-values/ansible_inventory \
     --user       $user
 
 ansible-playbook migrate-db.yml                                    \
-    --extra-vars cluster=$cluster                                  \
+    --extra-vars "cluster=$cluster cluster_type=$cluster_type"     \
     --inventory  ../../concent-deployment-values/ansible_inventory \
     --user       $user
 ```
 
+The above assumes that the `$cluster_type` shell variable is set to the name of the cluster type `(control/storage)` which defines what database is created.
+
```

Comment:
This line will not be necessary after rewording I suggested above.

Reply:
Done.